frontend/src/components/MXEndpoint.js

```diff
@@ -55,14 +88,46 @@ function MXEndpoint({
                 {title}
               </Text>
             </div>
-            <div className="mt-8">
+            {showNotice && (
+              <Text as="ParagraphSmall" color="secondary" tag="p">
+                Notice: The first two parts were completed automatically in the widget because it was configured to {getNotice()}
+              </Text>
+            )}
+            <div className="mt-8 flex-align">
               <span className='mr-8'>
                 {requestLabel()}
               </span>
               <Text className='code-text' as="ParagraphSmall" color="primary" tag="span">
                 {requestUrl}
               </Text>
+              {loadingStatusLabel(status, 0)}
             </div>
+            {
+              status >= 1 && (
+                <div className="mt-8 flex-align status-step">
+                  <span className='mr-8'>
+                    <Tag title="Get" variant="success" />
+                  </span>
+                  <Text className='code-text' as="ParagraphSmall" color="primary" tag="span">
+                    {'/users/{user_guid}/members/{member_guid}/status'}
+                  </Text>
+                  {loadingStatusLabel(status, 1)}
+                </div>
+              )
+            }
+             {
+              status >= 2 && (
+                <div className="mt-8 flex-align status-step">
+                  <span className='mr-8'>
+                    <Tag title="Get" variant="success" />
+                  </span>
+                  <Text className='code-text' as="ParagraphSmall" color="primary" tag="span">
+                    {finalDataUrl}
+                  </Text>
+                  {loadingStatusLabel(status, 2)}
+                </div>
+              )
+            }
```

Comment:
I noticed in these two sections are rendering the same type of content 1 after the other.  We could consider creating a tiny component to reduce some duplicate code if we wanted to and it's worth the time.

The other suggestion I have is to clarify the meaning of the status numbers.  Status of 0, 1, 2 take some work to discover.  I'd suggest using an object map or constants with good names to help make understanding them  a little easier.